codemon/CodemonReg.py

```diff
@@ -0,0 +1,28 @@
+import sys
+import os
+import ctypes
+from clint.textui import colored
+
+ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+
+def codemonReg():
+    print(ROOT_DIR)
+    path = os.path.join(ROOT_DIR, "codemon")
+    if not os.path.exists(path):
+        os.makedirs(path)
+        FILE_ATTRIBUTE_HIDDEN = 0x02
+        ret = ctypes.windll.kernel32.SetFileAttributesW(path, FILE_ATTRIBUTE_HIDDEN)
```

Comment:
This will work only on Windows systems but not on linux, please use the conventions paramsiddharth has used to allow codemon to work on both Windows and Unix systems.